samples/Erlang/release.es

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env escript
```

Comment:
Erm, and as for the shebang removal (just reread your question now... ugh, sorry), I didn't want to skew the results by telling Linguist to expect a shebang in the header of any Erlang `.es` files. Not sure if that actually has any bearing on classification, but since not every executable EScript file will have the shebang, I figure it wouldn't hurt to.